### PR TITLE
Upgrade to kehaar 0.8.1

### PR DESCRIPTION
**Pull request by tie-rack**

### CODE REVIEW

File: resources/config.edn [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
         :port #resource-config/edn #resource-config/env "SFTP_PORT"
         :username #resource-config/env "SFTP_USERNAME"
         :password #resource-config/env "SFTP_PASSWORD"}
- :timeouts {:schedule-message 20000
-            :delete-message 20000}
- :page-count-directory-format "%d_Page"
- :threads {:mailing-send #resource-config/edn #resource-config/env "MAIL_WORKS_MAILING_SEND_THREADS"
-           :mailing-schedule #resource-config/edn #resource-config/env "MAIL_WORKS_MAILING_SCHEDULE_THREADS"
-           :mailing-unschedule #resource-config/edn #resource-config/env "MAIL_WORKS_MAILING_UNSCHEDULE_THREADS"}}
+ :timeouts {:schedule-message 40000
```
Comment (ddellacosta) [inline resources/config.edn]: These can go away, can't they? I didn't see them used anywhere other than in the queue config.

File: resources/config.edn [hunk 2]
```diff
@@ -1,18 +1,40 @@
...
+                       :f mail-works.handlers.unschedule/unschedule-mailing
+                       :response true
+                       :threads #resource-config/edn #resource-config/env "MAIL_WORKS_MAILING_UNSCHEDULE_THREADS"
+                       :prefetch-limit #resource-config/edn #resource-config/env "MAIL_WORKS_MAILING_UNSCHEDULE_THREADS"}]
+
+                     :external-services
+                     [{:queue "scheduled-message-works.message.schedule"
+                       :channel mail-works.channels/schedule-message
+                       :response true
+                       :timeout 40000}
```
Comment (ddellacosta) [inline resources/config.edn]: Why the bump for these two, any reason in particular?

Comment (tie-rack) [inline resources/config.edn]: In the process of making these PRs, I keep being asked to bump timeouts to 40 seconds. Thought I'd take care of it before I was asked again.

File: resources/config.edn (hunk 1 above)
Comment (tie-rack) [inline resources/config.edn]: You're right. Out they go.

File: resources/config.edn (hunk 2 above)
Comment (ddellacosta) [inline resources/config.edn]: Gotcha. Just wasn't sure, as I have less familiarity with some of this code than others. Thanks.

**Comment by ddellacosta** [comment]
I was trying to type "thumbs up" and a taco emoji came up so I just left it there, 'cause, that's pretty good too I think

**Pull request merged**